src/Core/Grid/Definition/Factory/AddressGridDefinitionFactory.php

```diff
@@ -289,7 +290,13 @@ protected function getBulkActions(): BulkActionCollectionInterface
                     ->setName($this->trans('Delete selected', [], 'Admin.Actions'))
                     ->setOptions([
                         'submit_route' => 'admin_addresses_delete_bulk',
-                        'confirm_message' => $this->trans('Delete selected items?', [], 'Admin.Notifications.Warning'),
+                        'confirm_message' => $this->trans('Are you sure you want to delete the selected item(s)?', [], 'Admin.Notifications.Warning'),
```

Comment:
Actually this whole code is present in a trait `BulkDeleteActionTrait` see `CmsPageDefinitionFactory` for an example

Reply:
done !